addons/common/functions/fnc_getArtilleryETA.sqf

```diff
@@ -0,0 +1,32 @@
+#include "script_component.hpp"
+/*
+ * Author: Kex
+ * Returns ETA to the target in seconds for given artillery unit based on
+ * target position and used magazine, -1 if target can't be hit.
+ * Also supports VLS.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Target position <ARRAY>
+ * 2: Ammo class <STRING>
+ *
+ * Return Value:
+ * ETA in seconds <NUMBER>
```

Comment:
Is it rounded seconds? As a user, I disliked seeing point something of a second from Achilles.

If it isn't rounded to no decimal point, then, in my opinion, it should be.

Reply:
I disagree. I don't see your point in enforcing an integer output.